graders/r/README.md

```diff
@@ -1,10 +1,31 @@
-## PrairieLearn Autograding R Docker Image
+## rocker-pl
 
+This is a Docker image working as an external grader to be paired with PrairieLearn.  It has been
+used in the [STAT 430](https://stat430.com) (2018-2020) and [STAT 447](https://stat430.com) (since
+2021) courses on _Data Science Programming Methods_ at the University of Illinois at
+Urbana-Champiagn. In a previous course iteration, a CentOS7-based container was used via Python. We
```

Comment:
*Champaign

Reply:
Ouch. Will fix. Thanks for spotting that.

Done in 3191a19 (after rebasing to origin/master)